pyseir/cli.py

```diff
@@ -164,13 +164,13 @@ def _write_pipeline_output(
         output_dir_path.mkdir()
 
     infection_rate_metric_df = pd.concat((p.infer_df for p in pipelines), ignore_index=True)
-    multiregion_rt = MultiRegionDataset.from_timeseries_df(infection_rate_metric_df)
+    multiregion_rt = MultiRegionDataset.from_geodata_timeseries_df(infection_rate_metric_df)
```

Comment:
I'm not sure I would expect this to be a geodata timeseries?

Reply:
I think infection_rate_metric_df has some static data, such as aggregation level so it is geodata and timeseries in one DataFrame.